### DISCUSSION TITLE
Is Chamilo able to mirror to a locally-hosted install?

### DISCUSSION THREAD

**Discussion by kd4e** [chamilo/chamilo-lms#5670] [General]
If I use A2Hosting for my public-facing Chamilo-LMS install is it designed to mirror to a locally-hosted (home) Chamilo-LMS install - as a backup?
I'd like the redundancy in case of a major hosting outage, de-platforming hassles, etc.
Thanks.

**Comment by Kaneda-1** [comment]
Yes, Chamilo can be installed locally like any other software.
You can use Xampp https://www.apachefriends.org/download.html on Linux, Windows or Mac.
Or install Apache, PHP, MySQL or MariaDB in the usual way.
